packages/plugin-version/sources/versionUtils.ts

```diff
@@ -19,8 +19,8 @@ export enum Decision {
 export type Releases =
   Map<Workspace, Exclude<Decision, Decision.UNDECIDED>>;
 
-export async function fetchBase(root: PortablePath) {
-  const candidateBases = [`master`, `origin/master`, `upstream/master`];
+export async function fetchBase(root: PortablePath, {baseRef}: {baseRef?: string | null}) {
+  const candidateBases = baseRef !== null ? [baseRef] : [`master`, `origin/master`, `upstream/master`];
```

Comment:
I'm thinking, what do you think about making it an array of refs instead, that would default to `['master', 'origin/master', 'upstream/master']` (what it currently is)? This way people who want to use a custom branch would be able to keep the behaviour that tries to take the origin into account if possible.

Reply:
I wanted to do that, but did not know how to use an array type. I see now I just need to add `isArray: true`. I will add this

Reply:
Hey Maël, I have updated the code and documentation accordingly. I've tested it and it works well. The only limitation is that using the environment variable will only allow a single ref to be set, not a huge deal imo. That could also be user error, I didn't take the time to see if there was a syntax to pass an array.